tests/core/index_management/test_get_settings.py

```diff
@@ -208,4 +206,176 @@ def test_custom_settings(self):
             retrieved_index = self.config.index_management.get_index(self.structured_custom_index.name)
             retrieved_settings = IndexSettings.from_marqo_index(retrieved_index).dict(exclude_none=True, by_alias=True)
             self.assertEqual(retrieved_settings, expected_structured_custom_settings)
-            
\ No newline at end of file
+
+    def test_index_settings_should_hide_openclip_model_internal_data_if_model_is_marqtune(self):
+        """Model name, dimensions, and model location should be hidden if model is marqtune
+        """
+        unstructured_marqtune_index = IndexSettings(
+            type=IndexType.Unstructured,
+            model='marqtune/model-id/release-checkpoint',
+            modelProperties= {
+                 "isMarqtuneModel": True,
+                 "name": "ViT-B-32",
+                 "dimensions": 512,
+                 "model_location": {
+                     "s3": {
+                         "Bucket": "marqtune-public-bucket",
+                         "Key": "marqo-test-open-clip-model/epoch_1.pt",
+                     },
+                     "auth_required": False
+                 },
+                 "type": "open_clip",
+            },
+            normalizeEmbeddings=False,
+            textPreprocessing=TextPreProcessing(splitLength=3, splitMethod=TextSplitMethod.Word, splitOverlap=1)
+        ).to_marqo_index_request('a' + str(uuid.uuid4()).replace('-', ''))
+
+        indexes = self.create_indexes([
```

Comment:
Do we clean up the index after the test run? Consider adding them to the `setUpClass` method.

Reply:
Moved the index creation to the setup method.